app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,36 @@
+class PasswordResetsController < ApplicationController
+  before_action :set_user_by_token, only: [:edit, :update]
+  def new
+
+  end
+  def create
+    if (user = User.find_by(email: params[:email]))
+      PasswordMailer.with(
+        user: user,
+        token: user.generate_token_for(:password_reset)
+      ).password_reset.deliver_later
+    end
+    redirect_to root_path, notice: "Check your email to reset your password"
+  end
+  def edit
```

Comment:
make sure to add EOL after each function. Fix this in other places as well. Thanks.
```suggestion
  end
  
  def edit
```

Reply:
added new line after the end of all function bhaiya